src/main/java/com/blackduck/integration/sca/upload/client/uploaders/UploaderFactory.java

```diff
@@ -64,13 +85,26 @@ public BinaryUploader createBinaryUploader(String urlPrefix, BinaryScanRequestDa
     /**
      * Construct the uploader for Container uploads.
      *
-     * @param urlPrefix Used to create {@link UploadRequestPaths}.
+     * @param urlPrefix
+     *            Used to create {@link UploadRequestPaths}.
      * @return the {@link ContainerUploader} created.
      */
     public ContainerUploader createContainerUploader(String urlPrefix) {
         return new ContainerUploader(uploaderConfig.getUploadChunkSize(), createFileUploader(urlPrefix), createUploadValidator());
     }
 
+    /**
+     * Construct the uploader for SCASS uploads.
+     *
+     * @return
```

Comment:
Add the return type to the javadoc.  `the {@link ScassUploader} created.`

Reply:
Added.